src/main/java/core/basesyntax/dao/jdbc/CarDaoJdbcImpl.java

```diff
@@ -0,0 +1,207 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.CarDao;
+import core.basesyntax.exceptions.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Car;
+import core.basesyntax.model.Driver;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class CarDaoJdbcImpl implements CarDao {
+    @Override
+    public Car create(Car car) {
+        String createString = "INSERT INTO cars (car_model, manufacturer_id"
+                + ") VALUES (?,?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(createString,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            statement.setString(1, car.getModel());
+            statement.setLong(2, car.getManufacturer().getId());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+            return car;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Unable to CREATE car in DB " + car, e);
+        }
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getCarString = "SELECT c.car_id, car_model AS model, c.manufacturer_id, "
+                + "manufacturer_name AS made_by, manufacturer_country AS country "
+                + "FROM cars c "
+                + "INNER JOIN manufacturers m "
+                + "ON c.manufacturer_id=m.manufacturer_id "
+                + "WHERE c.deleted=false AND c.car_id=?;";
+        String getCarDrivers = "SELECT cd.driver_id, d.driver_name AS driver_name, "
+                + "d.driver_license AS license "
+                + "FROM cars_drivers cd "
+                + "INNER JOIN drivers d "
+                + "ON cd.driver_id=d.driver_id "
+                + "WHERE car_id=? AND d.deleted=false;";
```

Comment:
let's create method getDrivers(). You can use it here and in getAll() method

Reply:
done